WebApiCoreSeed.UnitTests/WebApiCoreSeed.UnitTests.csproj

```diff
@@ -5,16 +5,25 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Remove="Resources\**" />

```

Comment:
Probably Resources folder is for non-code files used in the tests, fixtures, input data, etc